app/templates/detail.hbs

```diff
@@ -1,243 +1,250 @@
 
-                        <td><VoteTableEntry
-                            @familyName={{mandatary.proponent.alias.familyName}}
-                            @firstName={{mandatary.proponent.alias.firstNameUsed}}
-                            @fraction={{mandatary.proponent.hasMembership.innerGroup.name}}
-                          /></td>
-                        <td>
-                          <VoteTableEntry
-                            @familyName={{mandatary.opponent.alias.familyName}}
-                            @firstName={{mandatary.opponent.alias.firstNameUsed}}
-                            @fraction={{mandatary.opponent.hasMembership.innerGroup.name}}
-                          />
-                        </td>
-                        <td>
-                          <VoteTableEntry
-                            @familyName={{mandatary.abstainer.alias.familyName}}
-                            @firstName={{mandatary.abstainer.alias.firstNameUsed}}
-                            @fraction={{mandatary.abstainer.hasMembership.innerGroup.name}}
-                          />
-                        </td>
-                      </tr>
+              {{#if this.hasVotes}}
+                <section class="c-accordion-holder">
+                  <AuBadge @icon="message" @skin="brand" />
+                  <AuAccordion
+                    @reverse={{true}}
+                    @iconOpen={{"nav-down"}}
+                    @iconClosed={{"nav-right"}}
+                    @buttonLabel={{"Stemming"}}
+                  >
+                    {{#each this.model.agendaItem.handledBy.hasVotes as |vote|}}
+                      <VoteOverview
+                        @numberOfAbstentions={{vote.numberOfAbstentions}}
+                        @numberOfOpponents={{vote.numberOfOpponents}}
+                        @numberOfProponents={{vote.numberOfProponents}}
+                      />
                     {{/each}}
-                  </:body>
-                </AuTable>
-              </AuAccordion>
-            </AuAccordion>
-          </section>
-        {{/if}}
-      </c.content>
-    </AuCard>
-    {{#if this.model.agendaItemOnSameSession}}
-      <AuCard
-        @shadow={{true}}
-        class="au-c-card au-u-margin-large au-u-flex"
-        as |c|
-      >
-        <c.header>
-          <h2 class="au-u-h3">
-              Meer agenda op {{this.model.agendaItem.session.dateFormatted}}
-          </h2>
-        </c.header>
-        <c.content class="au-u-flex au-u-flex--column">
+
+                    <div class="c-accordion-holder c-accordion-holder--top">
+                      <AuAccordion
+                        @reverse={{true}}
+                        @iconOpen={{"nav-down"}}
+                        @iconClosed={{"nav-right"}}
+                        @buttonLabel={{"Wie stemde?"}}
+                        @subtitle="Meer informatie over stemmers en hun partijen"
+                      >
+                        <AuTable>
```

Comment:
Not sure if all mobile fixes are planned for this PR, but it splits into multiple tables on small devices.